cloe_shell.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
consider using the shebang /usr/bin/env bash

Reply:
I see this for python but have rarely seen it for Bash. Is it important for Bazel specifically?

There is: https://unix.stackexchange.com/questions/206350/what-is-the-difference-if-i-start-bash-with-bin-bash-or-usr-bin-env-bash